### PR TITLE
Add work-accident to w3id.org/italia

### PR DESCRIPTION
**Pull request by InailUfficio5**
Add work-accident to w3id.org/italia for redirecting to ontologies and open data of INAIL (the National Institute for Insurance against Accidents at Work)

### CODE REVIEW

File: italia/work-accident/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Repository (italy/work-accident) for redirecting to ontologies and open data of INAIL (the National Institute for Insurance against Accidents at Work)
```
Comment (davidlehn) [inline italia/work-accident/README.md:1]:
```suggestion
Repository (italia/work-accident) for redirecting to ontologies and open data of INAIL (the National Institute for Insurance against Accidents at Work) 

```

**Comment by davidlehn** [comment]
@bfabio @mfortini @giorgialodi You all are listed as contacts for `/italia/`.  Is it ok to add this `work-accident` sub-path?

**Comment by mfortini** [comment]
Yes, sure. Thank you for your message.

**Comment by bfabio** [comment]
@davidlehn yes, thanks

File: italia/work-accident/README.md (hunk 1 above)
Comment (InailUfficio5) [reply italia/work-accident/README.md:1]: Suggested change can be applied.

Comment (TallTed) [reply italia/work-accident/README.md:1]: fixed in #3595

**Pull request merged**